tests/unit/services/cookies-test.js

```diff
@@ -23,10 +23,16 @@ describe('CookiesService', function() {
       }).to.throw();
     });
 
-    it('throws when the HTTP only option is set', function() {
-      expect(() => {
-        this.subject().write(COOKIE_NAME, 'test', { httpOnly: true });
-      }).to.throw();
+    it('throws when the HTTP only option is set in a browser', function() {
```

Comment:
There's dedicated `describe`s for both FastBoot and the browser actually: [here](https://github.com/simplabs/ember-cookies/blob/45354e87766b5c75bbfc57262a1baf8305438505/tests/unit/services/cookies-test.js#L100) and [here](https://github.com/simplabs/ember-cookies/blob/45354e87766b5c75bbfc57262a1baf8305438505/tests/unit/services/cookies-test.js#L430). Can this be split up and moved into the respective `describe`s?